staging/src/k8s.io/kubectl/pkg/cmd/create/create_service.go

```diff
@@ -17,13 +17,22 @@ limitations under the License.
 package create
 
 import (
-	"github.com/spf13/cobra"
+	"context"
+	"fmt"
+	"strings"
 
+	"github.com/spf13/cobra"
```

Comment:
Try to keep imports sepearated into 3 groups:
1. built-in
2. 3-rd party
3. k8s imports.
See other files for how this is done.